packages/core/addon/services/data.js

```diff
@@ -196,6 +196,11 @@ class Card {
 
   get adoptedFromId() {
     let internal = priv.get(this);
+
+    if (internal.id === baseCard) {
+      return null;
```

Comment:
I think this needs a test, like `assert.equal(parent.adoptedFromId, null);` added to https://github.com/cardstack/cardstack/blob/master/packages/core/tests/unit/services/data-test.js#L694

Reply:
👍